Kernel/Locking/Mutex.cpp

```diff
@@ -230,8 +232,10 @@ void Mutex::unblock_waiters(Mode previous_mode)
         m_mode = Mode::Shared;
         for (auto& thread : m_blocked_threads_list_shared) {
             auto requested_locks = thread.unblock_from_lock(*this);
-            auto set_result = m_shared_holders.set(&thread, requested_locks);
-            VERIFY(set_result == AK::HashSetResult::InsertedNewEntry);
+            m_shared_holders += requested_locks;
+#if LOCK_SHARED_UPGRADE_DEBUG
+            m_shared_holders_map.set(&thread, requested_locks);
```

Comment:
This drops the `set_result == AK::HashSetResult::InsertedNewEntry` verify, is that no longer true, just not useful, or was it just forgotten?

Reply:
It is still true, it's just that the VERIFY was used to verify that the `unblock_waiters` method was working correctly, it had nothing to do with the shared -> exclusive upgrade check, but since it's behind a debug flag anyways why not adding more checking I guess, I'll add it back.